overlord/snapstate/snapmgr.go

```diff
@@ -420,6 +435,9 @@ func (m *SnapManager) doDiscardSnap(t *state.Task, _ *tomb.Tomb) error {
 			newSeq = append(newSeq, si)
 		}
 		snapst.Sequence = newSeq
+		if snapst.Current == ss.Revision {
```

Comment:
but if we put back the concept of disable/enable, we wouldn't know what "current" was across that, so maybe not a good idead